shellguide.md

```diff
@@ -196,7 +196,7 @@ Example:
 # Arguments:
 #   None
 #######################################
-function cleanup() {
+cleanup() {
```

Comment:
i've posted a CL internally to strip the `function` keywords from these few examples